internal-registry/che-incubator/command-line-terminal-dev/nightly/meta.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v2
+publisher: che-incubator
+name: command-line-terminal
+version: nightly
+type: Che Editor
+displayName: Command Line Terminal
+title: Command Line Terminal
+description: Command Line Terminal provides the ability to start a terminal inside
+  the OpenShift Console.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/eclipse/che-machine-exec/
+firstPublicationDate: "2020-05-13"
+category: Other
+spec:
+  endpoints:
+    - name: command-line-terminal
+      public: true
+      targetPort: 4444
+      attributes:
+        protocol: http
+        type: ide
+        discoverable: false
+        secure: true
+        cookiesAuthEnabled: true
+  containers:
+    - name: command-line-terminal
+      image: "quay.io/eclipse/che-machine-exec:nightly"
+      command: ["/go/bin/che-machine-exec",
+                "--authenticated-user-id", "$(CHE_WORKSPACE_CREATOR)",
+                "--idle-timeout", "$(CHE_WORKSPACE_IDLE_TIMEOUT)"]
```

Comment:
Do you think we should exclude that from the internal registry for released dockerimage?
Or it's safe enough to provide it as well?

Reply:
I think it's okay to provide as well -- someone could always deploy a custom registry and use that to run whatever plugins they want. If it's not included in the release, then testing anything that depends on this change would require building and deploying the operator (which could be tedious for e.g. console changes). 

If there's a security concern, though, I'm okay with leaving it out.

Reply:
Let's figure out later.